Brickwall/__init__.py

```diff
@@ -1,3 +1,51 @@
 from flask import Flask
+from flask_restplus import Api, Resource, fields
 
-app = Flask(__name__)
\ No newline at end of file
+flask_app = Flask(__name__)
+app = Api(app = flask_app,
+    version="0.0",
+    title= "Brickwall",
+    description="The Backend of the Brickwall webapp")
+
+name_space = app.namespace('main', description='Main APIs')
+
+@name_space.route('/api/v1/location/<location_id>')
+class Location(Resource):
+    def get(self, location_id):
+        print("location get")
+        return {"location": location_id}
+    def post(self, location_id):
+        print("Location put")
+    def put(self):
```

Comment:
You're going to want `location_id` on every method that will use it, so, everything except `PUT`, where `PUT` would be for creating a new one